### PR TITLE
Added functionality to add icon to the center of the qr code.

### PR DESCRIPTION
**Pull request by AlexGipp**

### CODE REVIEW

File: src/SkiaSharp.QrCode/QRCodeRenderer.cs [hunk 1]
```diff
@@ -19,8 +19,9 @@ public class QRCodeRenderer : IDisposable
         /// <param name="area">The area.</param>
         /// <param name="data">The data.</param>
         /// <param name="qrColor">The color.</param>
-
-        public void Render(SKCanvas canvas, SKRect area, QRCodeData data, SKColor? qrColor)
+        /// <param name="icon">The icon.</param>
+        /// <param name="iconSizePercent">The icon size.</param>
+        public void Render(SKCanvas canvas, SKRect area, QRCodeData data, SKColor? qrColor, SKBitmap icon = null, int iconSizePercent = 15)
         {
```
Comment (guitarrapc) [inline src/SkiaSharp.QrCode/QRCodeRenderer.cs:25]: How about adding `IconData` represent icon info?

```csharp
public class IconData
{
    public SKBitmap Icon { get; set; }
    public int IconSizePercent { get; set; } = 15;
}
```

So signature will be like this?
```csharp
public void Render(SKCanvas canvas, SKRect area, QRCodeData data, SKColor? qrColor, IconData icon)
```

Comment (Alex-451) [reply src/SkiaSharp.QrCode/QRCodeRenderer.cs:25]: Sounds like a great idea, i just implemented that 👍 
Thanks for the suggestion!

Comment (Alex-451) [reply src/SkiaSharp.QrCode/QRCodeRenderer.cs:25]: btw should i move the ECCLevel class into the newly created models folder?

File: src/SkiaSharp.QrCode/QRCodeRenderer.cs [hunk 2]
```diff
@@ -45,15 +45,15 @@ public void Render(SKCanvas canvas, SKRect area, QRCodeData data, SKColor? qrCol
                     }
                 }
 
-                if (icon != null)
+                if (iconData.Icon != null)
```
Comment (guitarrapc) [inline src/SkiaSharp.QrCode/QRCodeRenderer.cs]:
```suggestion
                if (iconData?.Icon != null)
```

File: src/SkiaSharp.QrCode/QRCodeRenderer.cs (hunk 1 above)
Comment (guitarrapc) [reply src/SkiaSharp.QrCode/QRCodeRenderer.cs:25]: No, moving ECCLevel to SkiaSharp.QrCode.Models namespace is breaking change.

**Comment by guitarrapc** [comment]
closed #7

File: src/SkiaSharp.QrCode/QRCodeRenderer.cs (hunk 1 above)
Comment (guitarrapc) [reply src/SkiaSharp.QrCode/QRCodeRenderer.cs:25]: no, moving ECCLevel class to SkiaSharp.QrCode.Models namespace is breaking change.

**Review by guitarrapc — APPROVED** [review]
LGTM

**Pull request merged**